### ISSUE TITLE
Memory error 

### ISSUE THREAD

**Issue by jemten** [mikaell/chromograph#11]
I get the following error when running chromograph using a wig file from tiddit. 
Any ideas?
```
../chromograph.py:358: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  fig, ax = plt.subplots(figsize=(8, .7))
```

**Comment by jemten** [comment]
My bad. It's merely a warning

**Comment by dnil** [comment]
Let’s reopen this to deal with the warning though. Either silence it as per its instructions or close after each.

**Comment by mikaell** [comment]
Fixed in latest commit (08392e7b).
